templates/flink.md

```diff
@@ -0,0 +1,62 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
```

Comment:
Please move this file to the right folder, not in templates.

Reply:
Move to the dev tools